### DISCUSSION TITLE
Minor UI suggestions

### DISCUSSION THREAD

**Discussion by hamensman** [libre-tube/LibreTube#826] [Ideas]
I'd like to offer some minor suggestions to the UI. If these things are already there, and I didn't notice because I'm an idiot, feel free to disregard. I will reference this screenshot below:
![libretube](https://user-images.githubusercontent.com/76445071/179760475-20b50cda-a0a8-4a67-9a23-1feb3c416711.jpeg)
So I search for something and tap on a video, the video appears, the I swipe down to show what you see here.

Let's say I want to resume watching the video (circled **red**). What I have to do is swipe up, which is fine. But I also think it would be intuitive to simply tap the area **too** (add tap in addition to swipe up, not replace).

And let's say I want to to go back to the video by tapping on the original link(?) (circled **blue**). At the moment, it restarts the video. But I think it would make more sense if it resumes back to the where you were in the video while it was playing. Or make that a toggle setting if you prefer.

**Comment by Silther** [comment]
Can this discussion be closed since the suggestions have been implemented?
